data/json/monsters/reptile_amphibian.json

```
@@ -56,7 +56,7 @@
     "armor_cut": 8,
     "vision_day": 3,
     "vision_night": 35,
-    "harvest": "animal_large_noskin",
+    "harvest": "mutant_animal_large_noskin",
```

Gator isn't a mutant, this isn't even a particularly large specimen.
```suggestion
    "harvest": "animal_large_noskin",
```

---

Whoops, good catch, I think I misread the description.